spec/webauthn/authenticator_attestation_response_spec.rb

```diff
@@ -55,6 +57,8 @@
     )
 
     expect(response.valid?(original_challenge, original_origin)).to eq(true)
+    expect(response.attestation_type).to eq(WebAuthn::AttestationStatement::Base::ATTESTATION_TYPE_SELF)
```

Comment:
I would try to avoid using internal constants for test expectations.
We want the test to fail if we accidentally change the constant value, right?